src/services/BucketService.ts

```diff
@@ -60,6 +60,10 @@ export class BucketService {
             this.logger.error(`Unable to delete bucket with token: "${bucket.bucketToken}"`);
             return false;
         }
+        const currentSession = this.$ctx?.request.session ?? null;
```

Comment:
`this.$ctx?.request.session` will always exist at this point. 

this should be `this.$ctx?.request.session?.bucket ?? null;`